src/main/java/com/crowdin/cli/commands/picocli/PreTranslateSubcommand.java

```diff
@@ -49,6 +49,9 @@ public class PreTranslateSubcommand extends ActCommandWithFiles {
     @CommandLine.Option(names = {"--plain"}, descriptionKey = "crowdin.list.usage.plain")
     protected boolean plainView;
 
+    @CommandLine.Option(names = {"--label"}, descriptionKey = "params.label", paramLabel = "...", order = -2)
```

Comment:
This is the wrong description for this parameter within this command. You need to create a new message

```properties
crowdin.pre-translate.label=Label identifier. Could be specified multiple times
```

Reply:
That makes sense. Added this description in the latest message.